### ISSUE TITLE
Deterministic printer output

### ISSUE THREAD

**Issue by lucas-manuel** [crytic/slither#1374]
### Describe the desired feature

When running:
`slither <file> --print function-summary`
and
`slither <file> --print vars-and-auth`

The order or values in the arrays changes between runs. Examples below:
![Screen Shot 2022-09-12 at 11 49 10 AM](https://user-images.githubusercontent.com/44272939/189698848-0ad9b211-d0e4-4023-9757-655d0b5f5e20.png)
![Screen Shot 2022-09-12 at 11 50 12 AM](https://user-images.githubusercontent.com/44272939/189699065-3bb7629b-4a62-49ab-98ac-117c9be53072.png)

The intention was to run slither printers and write to files when updating submodules to check diffs, but since the order of the outputs in the arrays in non-deterministic, the diffs are too large to be usable.

Easiest solution in my opinion would be to simply alphabetically order the array values.

**Comment by Abhimanyu121** [comment]
I can take this up

**Comment by aviggiano** [comment]
This issue should be closed. I was looking for a `good first issue` and came across this one, but it is no longer relevant after #1513
